payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberConfig.kt

```diff
@@ -3,25 +3,44 @@ package com.stripe.android.ui.core.elements
 import androidx.compose.ui.text.input.KeyboardCapitalization
 import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.text.input.VisualTransformation
+import com.stripe.android.CardBrandFilter
 import com.stripe.android.CardUtils
 import com.stripe.android.model.CardBrand
 import com.stripe.android.uicore.elements.TextFieldState
 import com.stripe.android.uicore.elements.TextFieldStateConstants
 import com.stripe.android.R as StripeR
 
-internal class CardNumberConfig : CardDetailsTextFieldConfig {
+internal class CardNumberConfig(
+    private val isCBCEligible: Boolean,
```

Comment:
Can you make this `isCardBrandChoiceEligible`? I don't think we shorten anywhere else.